release.py

```diff
@@ -6,7 +6,7 @@
 
 AZURE_STORAGE_CONNECTION_STRING = os.environ.get(
     'AZURE_STORAGE_CONNECTION_STRING')
-BLOB_MSSQL_CLI_DAILY_CONTAINER_NAME = 'daily/mssql-cli'
+BLOB_MSSQL_CLI_DAILY_CONTAINER_NAME = 'daily/whl'

```

Comment:
I though we needed the package name in the path for pip. Please confirm it works with the --extra-index-url parameter.